apollo-gradle-plugin/src/main/groovy/com/apollographql/android/gradle/ApolloPlugin.groovy

```diff
@@ -42,7 +42,7 @@ class ApolloPlugin implements Plugin<Project> {
     project.getGradle().addListener(new DependencyResolutionListener() {
       @Override
       void beforeResolve(ResolvableDependencies resolvableDependencies) {
-        compileDepSet.add(project.dependencies.create("com.apollographql.android:api:${VersionKt.VERSION}"))
+        compileDepSet.add(project.dependencies.create("com.apollographql.android:api:0.2.1"))
```

Comment:
VersionKt.VERSION would resolve to 0.2.2-SNAPSHOT which we don't have yet so the build would fail.

We'd need another PR to change that after the snapshots are uploaded.